ui/pages/nightlies.js

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+
+import { request, gql } from 'graphql-request';
+
+import { Grid, IconButton, Stack, Tooltip, Typography } from '@mui/material';
+import { Circle } from '@mui/icons-material';
+
+const buildStatusColorMap = {
+  FAILED: 'error',
+  RUNNING: 'warning',
+  SUCCESS: 'success',
+  default: 'warning.dark' // WTF is going on colour!
+};
+
+const buildStatusColor = status => buildStatusColorMap[status] || buildStatusColorMap.default;
+
+const BuildStatus = ({ nightlies }) => {
+  const openNightlyClick = item => window.open(`https://gitlab.com${item.path}`, '_newtab');
+
+  return (
+    <>
+      <Typography variant="h4">Nightlies</Typography>
+      <Stack>
+        {Object.values(nightlies)
+          .reverse()
+          .map(monthlyNightlies => {
+            const date = new Date(monthlyNightlies[0].startedAt);
+            return (
+              <React.Fragment key={date.getMonth()}>
+                <Typography variant="h6">{date.toLocaleString('default', { month: 'long' })}</Typography>
+                <Grid container direction="row" justifyContent="flex-start" alignItems="flex-start">
+                  {monthlyNightlies.reverse().map(item => (
+                    <Stack key={item.path} alignItems="center">
+                      <Tooltip
+                        arrow
+                        title={
+                          <>
+                            {new Date(item.startedAt).toLocaleString()}
+                            {Object.entries(item.testReportSummary.total).map(([name, value]) => (
+                              <Stack direction="row" justifyContent="space-between" key={name}>
+                                <b>{name}</b>
+                                <div>{Math.ceil(value)}</div>
```

Comment:
Q: Why? would we ever get fractions here?

Reply:
the duration comes in a fraction 👌